helm/polaris/values.yaml

```
@@ -528,8 +528,14 @@ persistence:
   relationalJdbc:
     # -- The secret name to pull the database connection properties from.
     secret:
-      # -- The secret name to pull the database connection properties from.
+      # -- The secret name to pull database connection properties from
       name: ~
+      # -- Database username for authentication
```

Can we be a bit more explicit about the meaning of the field? E.g.

```suggestion
      # -- The secret key holding the database username for authentication
```